### PR TITLE
Sc parks nphotos

### PR DESCRIPTION
**Pull request by shaibird**
**### Description**

Added create, update, and delete functions for photos and parks. Also added the ability to get photos by park and photos by users. 

Fetch the sc-parksNphotos branch and switch to it
Start/restart debugger
Send a POST request to Parks (e.g. http://localhost:8088/parks)
post body: 
    {
        "name": "The park down the road",
        "history": "It's wayyy down there. Just keep driving",
        "city": "Backwoods",
        "state": "Tennessee",
        "longitude": -115.060294,
        "latitude": 34.060442
    }
    
Verify 201 status code and new Park in list of Parks
If a post request is made with incomplete information, "missing information" will display in the body =

Send a PUT request to http://localhost:8088/parks/{new_post_id}
with post body: 
{
    "name": "The park down the road",
    "history": "It's wayyy down there. Just keep driving more driving",
    "city": "Backwoods",
    "state": "Kentucky",
    "longitude": -115.060294,
    "latitude": 34.060442
}

Verify changes to that user id with a GET request. PUT requests default to 204 status

Send a DELETE request to http://localhost:8088/parks/{new_post_id}

Verify 204 status code and the user is no longer in the Parks list

Repeat process for:
http://localhost:8088/photos 
post body: {
    "url": "pretend I'm an image url",
    "user_id": 4,
    "park_id": 5
}

**To get photos by user_id**

Send a GET request to http://localhost:8088/photos?user_id=11

Verify status code is 200 and that the returned body only contains images from specified user. 
I need to correct and change status if the user has not uploaded images. 

**To get photos by park_id** 

Send a GET request to http://localhost:8088/photos?park_id=2
Verify status code is 200 and that the returned body only contains images from specified park. 
I need to correct and change status if the user has not uploaded images.

### CODE REVIEW

File: views/__init__.py [hunk 1]
```diff
@@ -1,10 +1,9 @@
 from .wildlife_views import get_all_wildlife, get_single_wildlife_type, get_wildlife_by_park_id, create_wildlife, update_wildlife, delete_wildlife
 from .natural_attractions_views import get_all_natural_attractions, get_single_natural_attraction, get_natural_attraction_by_park_id, create_natural_attraction, update_natural_attraction, delete_natural_attraction
-from .amenity_views import get_all_amenities, get_all_amenity_types, get_amenity_by_id, get_amenity_type_by_id, delete_amenity_type, delete_park_amenity, create_amenity_type, create_park_amenity, update_amenity_type, update_park_amenity, get_amenities_by_park_id
-from .park_views import get_all_parks, get_single_park
-from .photo_views import get_all_photos, get_single_photo
 from .blog_views import get_all_blogs, get_single_blog, get_blogs_by_user_id, get_blogs_by_park_id, create_blog, update_blog
-from .user_views import get_all_users, get_user_by_id, get_user_favorite_by_id, get_all_user_favorites
```
Comment (vanessaspear) [inline views/__init__.py:7]: It looks like .user_views imports might have gotten deleted.  Can you check?

File: views/park_views.py [hunk 2]
```diff
@@ -48,3 +48,65 @@ def get_single_park(id):
     park = Parks(data['id'], data['name'], data['history'], data['city'], data['state'], data['latitude'], data['longitude'])
 
     return park.__dict__
+
+def create_park(park):
+    """creating a post request to create a park. Checks if all values are included in post, returns error"""
+    with sqlite3.connect("./national_park.sqlite3") as conn:
```
Comment (vanessaspear) [inline views/park_views.py:54]: Recommend using the create_resource function from sql_helper.py

**Review by mvdutta — APPROVED** [review]
Looks great!

**Review by vanessaspear — CHANGES_REQUESTED** [review]
I reviewed the parks updates, and everything passed the tests.  Only 2 comments for review.  Nice job!

File: views/park_views.py (hunk 2 above)
Comment (shaibird) [reply views/park_views.py:54]: Ah! I'll work on modifying in my next bunch of changes. Not enough brain power to make those changes right now and try to work it out the error message in the event of missing info.

File: views/__init__.py (hunk 1 above)
Comment (shaibird) [reply views/__init__.py:7]: I still have them! The merge conflicts I had earlier created two lines of .user_views, one was deleted.

**Comment by vanessaspear** [comment]
Approved

**Pull request merged**